service-runner/src/main/resources/profiles/default/config.env

```diff
@@ -38,6 +40,7 @@ sep38.enabled=true
 sep24.enabled=true
 sep24.interactive_url.base_url=http://localhost:8091/sep24/interactive
 sep24.more_info_url.base_url=http://localhost:8091/sep24/transaction/more_info
+custody.type=none
```

Comment:
We should default `custody.type` to `none`.

Reply:
Fixed in https://github.com/stellar/java-stellar-anchor-sdk/pull/1082